internal/config/config.go

```diff
@@ -64,10 +67,12 @@ type API struct {
 }
 
 type EKS struct {
-	AccountID   string        `mapstructure:"account_id"`
-	Region      string        `mapstructure:"region"`
-	ClusterName string        `mapstructure:"cluster_name"`
-	APITimeout  time.Duration `mapstructure:"api_timeout"`
+	AccountID                         string             `mapstructure:"account_id"`
+	Region                            string             `mapstructure:"region"`
+	ClusterName                       string             `mapstructure:"cluster_name"`
+	APITimeout                        time.Duration      `mapstructure:"api_timeout"`
+	APINodeLifecycleDiscoveryDisabled *bool              `mapstructure:"api_node_lifecycle_discovery_disabled"`
```

Comment:
would definitely add a log for this saying that discovery is disabled, we won't be able to identify spots  and everything is considered on-demand